lang/go/go.talon

```diff
@@ -194,3 +269,19 @@ swipe [<user.text>] [over]:
     key("right")
     insert(", ")
     insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
+
+type <user.text> struct:
+    insert("type ")
+    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
+    insert(" struct {")
+    key("Enter")
+
+receiver function <user.text> for <user.text>:
+    insert("func (")
+    insert(user.formatted_text(text_2, "ABBREVIATION"))
+    insert(" *")
+    insert(user.formatted_text(text_2, "PUBLIC_CAMEL_CASE"))
+    insert(") ")
+    insert(user.formatted_text(text_1, "PUBLIC_CAMEL_CASE"))
+    insert("() {")
+    key("Enter")
```

Comment:
This feels like it should be done using snippets.  Not sure it makes sense to have something this complex built out of `insert`s.  I'd be inclined to just leave this one out and then at some point maybe we have a separate set of common snippets.  I'm also a bit hesitant about the `ABBREVIATION` formatter here